### PR TITLE
Add initial LOD functionality.

### PR DESCRIPTION
**Pull request by digisomni**
Follows the following conventions. See docs too: https://doc.babylonjs.com/features/featuresDeepDive/mesh/LOD

```[name]_LOD{ level }_{ mode }_{ override }```

Levels are:
* 0
* 1
* 2
* 3
* HIDE

e.g. `LODHIDE`

Modes are (optional):
* AUTO (not enabled right now)
* SIZE (based on the amount of the screen the object covers)
* DISTANCE

Overrides are (optional):
* if using distance mode: 0 - infinity (any number)
* if using size mode: Must be a number between 0.0 and 1.0, represents a percentage of 100%.

P.S. Need to fix up linting, it gets hecka annoying but it was inactive before.

### CODE REVIEW

**Comment by Aitolda** [comment]
Got our first BUG.

It doesn't currently respect the position of the LOD objects within the GLB, instead moving them to the 0,0,0 point.
100m offset
Note in the image above, the models are 100 meters from 0,0,0

https://github.com/vircadia/vircadia-web/assets/21160271/4f2a80fb-fd80-4ca5-83b6-120618078acb


Now if you note the position of the objects in this video, I've added the little X to note the 0,0,0 of the GLB (raised slightly so it wasn't in the floor, hence no Y value, just X and Z shown). In fact the models should be 100 meters away on the X from this point.

**Comment by Aitolda** [comment]
New Bugs.

1. having a second copy of these objects in the GLB doesn't seem to work.  Note this is the default naming scheme for copies in blender, The second set will still have the position bug, but as far as I can tell they (the second set) are all visible at all times.  This is likely due the the .xxx suffix not yet being accounted for.  Example below.

91e48400c448931a009e166241a22298

2. Also, rotation and scale (set in blender) from within the GLB seems to be ignored on both sets of objects just like the position, both the semi-working set and the non-working set. In blender I have both sets placed far away from  the zero point and rotated and scaled randomly, but here is how they look in engine.

0837ee5799ababa357a798e6103a665b

So basically, my take away is that in the process of enabling LOD we are somehow no longer pulling rotation, position of scale of objects within said GLB.  Now if I rotate the GLB itself from that native that rotation is being respected, but all the internal object transforms are being ignored. 

To shake things up I added a regular (non-LOD) cube to the glb file and confirmed that it's rotation and position are indeed being honored still.  You can see it floating in the sky instead of being pulled to 0,0,0 like the LOD objects.

9655258145e3ebe0e64ac32d72169e00

**Comment by Aitolda** [comment]
> Follows the following convention:
> 
> `[name]_LOD{ level }_{ mode }_{ override }`
> 
> Levels are:
> 
> * 0
> * 1
> * 2
> * 3
> * HIDE

Can you explainhow to use HIDE?  Is this supposed to be the cull distance?  First I tried
9401cf9da201b9fe6407e64d9248a1eb

And although that doesn't really make sense since this requires an extra object/mesh, but also LOD3 just kept visible to infinity, so I image that wasn't the intended use.

Then I tried
463effbf27256f626c26e9f9d4eeebdb
but this (not surprisingly) didn't work.

**Comment by Aitolda** [comment]
Not sure what exactly size is supposed to do, but this as written below had no visible impact
b71ddbaa9f466b469d01d39a6645ac23
